R/recorder.R

```diff
@@ -1,9 +1,12 @@
 #' Launch test event recorder for a Shiny app
 #'
-#' @param app A \code{\link{ShinyDriver}} object, or path to a Shiny application.
+#' @param app A \code{\link{ShinyDriver}} object, or path to a Shiny
+#'   application.
 #' @param save_dir A directory to save stuff.
+#' @param load_test A boolean that determines whether or not the resulting test
+#'   script should be appropriate for load testing
 #' @export
-recordTest <- function(app = ".", save_dir = NULL) {
+recordTest <- function(app = ".", save_dir = NULL, load_test = FALSE) {
```

Comment:
`load_test` doesn't quite seem right to me... maybe `load_mode`? (And same for stuff down below.)